pkg/validator/container.go

```diff
@@ -38,7 +38,7 @@ type ContainerValidation struct {
 //       relevant podSpec in order to check certain aspects of a containerSpec.
 //       Perhaps there is a more ideal solution instead of attaching a parent
 //       podSpec to every container Validation struct...
-func ValidateContainer(container *corev1.Container, parentPodResult *PodResult, controllerName string, conf *config.Configuration, isInit bool) ContainerResult {
+func ValidateContainer(container *corev1.Container, parentPodResult *PodResult, conf *config.Configuration, controllerName string, controllerType config.SupportedController, isInit bool) ContainerResult {
```

Comment:
More of a golang question here, is there a reason the argument order here needed to be changed?

Reply:
Definitely wasn't necessary. I merged in the change that added the `controllerType` parameter, and realized `controllerType` and `controllerName` should probably be next to each other.